terraform/gce-vm/startup-script.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+
+# Update and upgrade APT reop
+export DEBIAN_FRONTEND=noninteractive
+apt-get -y update
+apt-get -y upgrade
```

Comment:
Are we sure that we want to update all packages on startup? This is going to make the build less reproducible...

Reply:
Without update all packages, will cause the error on installation of some packages in later steps. This could be improved by updating newer version of OS image itself in future.